docs/UserGuide.adoc

```diff
@@ -151,6 +151,23 @@ Views all details of the 2nd person in the address book.
 `viewall 1` +
 Views all details of the 1st person in the results of the `find` command.
 
+== Edit name : `edit_name`
+
+Change the name of a person. +
+Format: `edit_name New Name n/Old Name`
+
+Examples:
+*`edit_name John Doe n/John Dee`
+
+== Edit Phone : `edit_phone`
+
+Change the phone number of a person. +
+Format: `edit_phone New Phone n/Old Phone`
```

Comment:
As mentioned in the tutorial, if there are multiple people with the same phone number, how will you decide which one to update?